### PR TITLE
Feature/apl 1238 repository AskOrder, BidOrder, Trade

### PR DESCRIPTION
**Pull request by silaev-firstbridge**
[APL-1238] AskOrder, BidOrder, Trade

### CODE REVIEW

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/trade/model/Event.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+ * Copyright © 2018-2020 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.core.trade.model;
+
+/**
+ * @author silaev-firstbridge on 4/14/2020
+ */
+public enum Event {
+    TRADE
```
Comment (andrii-firstbridge) [inline apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/trade/model/Event.java:11]: We have about 12 classes which have enum  Event. (Shuffling.Event, CurrencyMint.Event, ....)
So in this case I think it make sense  rename this class on EventTrade for example, to use second part of the name as class name where this event from.

File: apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/AccountEndpointTest.java [hunk 2]
```diff
@@ -51,6 +56,7 @@
 import static org.mockito.Mockito.mock;
 
 @EnableWeld
+@Disabled
```
Comment (andrii-firstbridge) [inline apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/AccountEndpointTest.java:59]: Do you want to fix it before merge?

Comment (silaev-firstbridge) [reply apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/AccountEndpointTest.java:59]: No, I'm gonna investigate a searching issue in Weld tests after code freeze

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/trade/model/Event.java (hunk 1 above)
Comment (silaev-firstbridge) [reply apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/trade/model/Event.java]: done

**Pull request merged**